utils/common/src/main/java/org/apache/brooklyn/util/time/Time.java

```diff
@@ -157,7 +157,7 @@ public static String makeTimeStringRounded(long t, TimeUnit unit) {
         return makeTimeStringNanoRounded(nanos);
     }
     public static String makeTimeStringRounded(Stopwatch timer) {
-        return makeTimeStringRounded(timer.elapsed(TimeUnit.MILLISECONDS), TimeUnit.MILLISECONDS);
+        return (timer == null) ? null : makeTimeStringRounded(timer.elapsed(TimeUnit.MILLISECONDS), TimeUnit.MILLISECONDS);
```

Comment:
Can you add `@Nullable`, useful when looking at the signature from the call site.